### ISSUE TITLE
Not scraping images or messages

### ISSUE THREAD

**Issue by Bum211** [datawhores/OF-Scraper#455]
Hi there, I downloaded the latest commit from yesterday. I finally got past authentication and the script to run regularly. However, I noticed that the script is scraping Video ok but not scraping any images on posts or in messages. I don't have any error or anything to share here as the script just runs normally. 

For example, there are models that I just sub today and it's not scraping anything but just the Profile and those images. Another model that I'm subbed to, is not scraping any images from posts but only videos.

![Screenshot 2024-08-26 202853](https://github.com/user-attachments/assets/9e5dd078-f79c-48b6-bd24-71e576d6f453)
![image](https://github.com/user-attachments/assets/2f53cb9d-f35a-4c2f-acd5-211e755dddd1)

There are two screenshots I could take when the script is running for this newly subbed today. However, there was nothing scrapped besides the Profile image and the header. 

Help, please? 

Thanks!
Bum

**Comment by Bum211** [comment]
Thank you for the quick reply. I tried both methods and it is still the same result. Only scrap the profile image and header. 

I deleted all the cache and the model cache in .data and even used the first method on the new sub or existing sub and it is still giving me the same result.

**Comment by datawhores** [comment]
Which commit exactly

**Comment by Bum211** [comment]
Hello, it was the one from 2 days ago. 

![image](https://github.com/user-attachments/assets/acd3888b-fa4f-4487-a761-27399ce6d2d0)

This commit actually scrape Vids that were otherwise didn't scrape before but it won't scrape any images

Thanks!

**Comment by lordoffools** [comment]
Seems to have just started happening for me a few days ago (I didn't notice until I did another run/pull today). Only pulling videos, and I'm not certain if it's all of them yet (need to do an audit). Definitely not pulling images, or any content in messages.

I can't be sure when it started, but it had to have started a few days ago / late last week.

**Comment by lordoffools** [comment]
Hmm, actually, I have an example where even videos weren't pulled. (New subscription as of today)

**Comment by nmlen** [comment]
Same issue

**Comment by datawhores** [comment]
Can you try maybe logging out and logging back in
Also send a log file

**Comment by Bum211** [comment]
[Ofscraper Log.txt](https://github.com/user-attachments/files/16761061/Ofscraper.Log.txt)

Here's the log file. Please let me know this suffice.

**Comment by datawhores** [comment]
Well according to the script all the downloads you want are already been marked as downloaded

You have to use --dupe --after 2000

**Comment by Bum211** [comment]
This is just one of the logs. It still doesn't help that I cannot scrape anything with a new sub. And I'm sorry I don't know what you mean by using dupe after 2000.

**Comment by Bum211** [comment]
[Ofscraper Log_New_Sub.txt](https://github.com/user-attachments/files/16761822/Ofscraper.Log_New_Sub.txt)

Here's the script I just ran with the new sub I subbed yesterday. Beside the script only getting her Profile Pic and Header, it didn't scrape anything else. This is how I found out the script wasn't scraping any images.

Thank you for looking into this!

**Comment by datawhores** [comment]
Not sure what happen but all the downloads you want are already marked as downloaded you need to force a rescan, and redownload with 
ofscraper --after 2000 --dupe

**Comment by Bum211** [comment]
Thank you for looking into it. Right now I'm running into another issue that 

![image](https://github.com/user-attachments/assets/723921de-9180-4b35-82a1-d691259b05de)

Can't even log the list of models. It was working ok about 2 hours ago.

**Comment by DADESUPER** [comment]
I'm gonna chime in and say that i too have noticed that in some cases some posts aren't being listed or downloaded at all, i tried purging the database and using dupe but same result. The only way i could get it to download the missing posts was running post_check mode and selecting every row in the table to be downloaded. Sometimes the difference is pretty significant, in one case nearly 100 posts were missing on a total of 140ish. At first i thought it may have been skipping duplicates but as it turns out this is not the case. Images were affected most but many videos were also affected by the bug

**Comment by 1138cl** [comment]
I personally found that even though I was running "pip install --upgrade ofscraper" that I was still running an old version that would only download videos. I downloaded the latest version from the release page and that appears to be download all images now.

**Comment by DADESUPER** [comment]
> I personally found that even though I was running "pip install --upgrade ofscraper" that I was still running an old version that would only download videos. I downloaded the latest version from the release page and that appears to be downloading all images now.

3.12.1 win build has the partial download outside of post_check mode for me, but only for some people for whatever reason, others are fine

**Comment by datawhores** [comment]
> I'm gonna chime in and say that i too have noticed that in some cases some posts aren't being listed or downloaded at all, i tried purging the database and using dupe but same result. The only way i could get it to download the missing posts was running post_check mode and selecting every row in the table to be downloaded. Sometimes the difference is pretty significant, in one case nearly 100 posts were missing on a total of 140ish. At first i thought it may have been skipping duplicates but as it turns out this is not the case. Images were affected most but many videos were also affected by the bug

Please open another issue with a log  of an effected user if possible

**Comment by datawhores** [comment]
The force reset of cache in 3.12.2 and beyond should fix some of these issues

**Comment by FAlbanni** [comment]
Yep sorry I know there is a lot of work behind this but this software is a bloated POS
Doesn't work at all

**Comment by hawktank** [comment]
I'm having a similar issue where the user has over 3000 posts but the scraper only grabs 1500 of them (images and videos), missing half the content.

**Comment by datawhores** [comment]
The script will remove duplicate files
Trying to download them will cause issues, because you will have to async processes trying to download potentially to the same location, depending on the users setting
